tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -20,15 +20,10 @@ parameters:
   type: boolean
   default: true
 
-- name: enable_linux_gpu_training_cu102
-  displayName: 'Whether Linux GPU Cuda 10.2 package is built.'
+- name: enable_linux_gpu_tests
```

Comment:
skipping the tests for the Python package build reduces build time from 3.5 hours to <1 hour. added a way to bypass them here. they are only skipped by build-perf-test-binaries-pipeline.yml now.